go/src/koding/klientctl/metrics/metrics.go

```diff
@@ -0,0 +1,76 @@
+package metrics
+
+import (
+	"context"
+	"path/filepath"
+
+	kitesconfig "koding/kites/config"
+
+	statsd "github.com/DataDog/datadog-go/statsd"
+)
+
+// Metrics wraps metric
+type Metrics struct {
+	bolt    *BoltQueue
+	Datadog *statsd.Client
+}
+
+// New creates new metrics collector.
+func New() (*Metrics, error) {
+	boltPath := filepath.Join(kitesconfig.KodingHome(), "metrics.bolt")
```

Comment:
If the metrics endpoint is going to be the same for prod & devel, then I agree it's less of a problem.